src/modules/rtpengine/doc/rtpengine_admin.xml

```diff
@@ -2028,14 +2028,17 @@ modparam("rtpengine", "control_cmd_tos", 144)
 			Default value is 0, legacy algorithm.
 		</para>
 		<para>
-			The values not falling into the range <quote>0-1</quote> are ignored.
+			The values not falling into the range <quote>0-2</quote> are ignored.
```

Comment:
The meaning of `2` should probably be mentioned in the paragraph just above (line ~2025). Otherwise LGTM.